Gemfile

```diff
@@ -20,11 +20,11 @@ if ENV['BEAKER_PE_PR_REPO_URL']
   gem lib, :git => "git@github.com:#{author}/#{lib}.git", :branch => ref
 end
 
-group :rubocop do
-  gem 'rubocop', '~> 1.12.0'
-  gem 'rubocop-performance'
-  gem 'rubocop-rake'
-  gem 'rubocop-rspec'
+group :tests do
```

Comment:
I liked the separate rubocop group. We don't do it today, but we can exclude RuboCop from our tests this way.

Reply:
done